src/cljc/rems/text.cljc

```diff
@@ -60,7 +60,8 @@
   (text (get states state :t.applications.states/unknown)))
 
 (def ^:private event-types
-  {:application.event/approved :t.applications.events/approved
+  {:application.event/accepted-licenses :t.applications.events/accepted-licenses
```

Comment:
`:application.event/licenses-accepted` would be more in line with how the other events are named.

Reply:
Indeed, but somewhere we probably made an error with this because now you have to remember in which order the words were ...

Reply:
No error. It follows the naming conventions of event sourced systems: commands are verbs in imperative mood, events are verbs in past tense. Typically it ends up with the verb changing location, for example "create order" and "order created".

"Accepted licenses" is not a verb, but a noun (participle + noun), short for "licenses which have been accepted". It's used in that sense in `:application/accepted-licenses`. Past verb for an event would be an active "applicant accepted licenses" or passive "licenses accepted". The active form could be used in situations where the events need to be handled differently based on who made the event ("applicant accepted licenses" vs "member accepted licenses"), but usually that's not needed and anyways the event contains information that who made the event, so the passive form is usually enough.